tests/test_column_level_knowledge_propagator.py

```diff
@@ -127,12 +126,13 @@ def test_inherit_column_level_knowledge():
             "quote": None,
         },
     }
+    if dbt_version >= Version("1.9.0"):
+        for key in expect.keys():
+            expect[key]["granularity"] = None
```

Comment:
It seems that a field called `granularity` will be added to dbt 1.9. I think this is probably to support the semantic layer.

- https://github.com/dbt-labs/dbt-core/issues/10376

I noticed this because my tests failed in CI.

- https://github.com/z3z1ma/dbt-osmosis/actions/runs/12248438885/job/34168045444